ArmVirtPkg/ArmVirtQemu.dsc

```diff
@@ -49,10 +49,10 @@
 
 !include NetworkPkg/NetworkDefines.dsc.inc
 
-!include ArmVirtPkg/ArmVirt.dsc.inc
-
 !include MdePkg/MdeLibs.dsc.inc
 
+!include ArmVirtPkg/ArmVirt.dsc.inc

```

Comment:
Not sure why - and I'll have to inspect the parsing rules - but MdeLibs was overriding the definition of ArmTrngLib that was added to ArmVirt.dsc.inc.

This allows the build to proceed